Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -314,6 +321,28 @@ private class GeometryEditorModel: ObservableObject {
         isUniformScale ? .uniform : .stretch
     }
     
+    init() {
+        let boundaryGraphic = Graphic(geometry: .boundary(), symbol: .polygon)
+        
+        let road1Graphic = Graphic(geometry: .road1(), symbol: .polyline)
+        
+        let road2Graphic = Graphic(geometry: .road2(), symbol: .polyline)
+        
+        let outbuildingsGraphic = Graphic(geometry: .outbuildings(), symbol: .multipoint)
+        
+        let houseGraphic = Graphic(geometry: .house(), symbol: .point)
+        
+        geometryOverlay.addGraphics([
+            boundaryGraphic,
+            road1Graphic,
+            road2Graphic,
+            outbuildingsGraphic,
+            houseGraphic
+        ])
```

Comment:
The existing geometries are there for showing the editing workflow. Can you add the identify operation so that editing existing geometries also works, as described in the implementation details? Thank you. See around `onIdentifyGraphicsOverlayStatusChanged` symbol in the Qt example.